### PR TITLE
PR to change the email in contributor.json

### PR DESCRIPTION
**Pull request by angelaizg**
#### 397e7c4255f22b3cd2f087066553cfd02279ccef
<pre>
PR to change the email in contributor.json

Reviewed by Aakash Jain and Jonathan Bedard.

* metadata/contributors.json:

Canonical link: <a href="https://commits.webkit.org/256503@main">https://commits.webkit.org/256503@main</a>
</pre>

https://github.com/WebKit/WebKit/commit/63877d86256904bff86b759e7778923bd20ee675

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/95978 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/5227 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/29019 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/105537 "Built successfully") | 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/5335 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/33989 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/88360 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/101360 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/101639 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/78/builds/3933 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/82583 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/30975 "Passed tests") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/9/builds/85782 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/87692 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/73808 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/39720 "Built successfully") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/73/builds/19220 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/37398 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/20561 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/41994 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/60/builds/43176 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/2166 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/43881 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/39821 "Passed tests") | |

### CODE REVIEW

**Comment by JonWBedard** [comment]
> #### [e9672bd](https://github.com/WebKit/WebKit/commit/e9672bd4096cb0f3d2a68e8be002e3760d5cc670)
> Changin email in contributors.json
> 
> Need the bug URL (OOPS!).
> Include a Radar link (OOPS!).
> 
> Reviewed by NOBODY (OOPS!).
> 

Need to remove the lines asking for bug and radar links. `git commit --amend --date=now` will do just fine, you don't need to change the content of the commit itself.

File: metadata/contributors.json [hunk 1]
```diff
@@ -756,7 +756,7 @@
    },
    {
       "emails" : [
-         "aizquierdogarcia@apple.com"
+         "a_izquierdogarcia@apple.com"
```
Comment (JonWBedard) [inline metadata/contributors.json]: My suggestion is to keep your old email as the second element in the list (I gave @Smackteo the same advice yesterday)

TLDR, if you keep your old email, tools will use your new email when printing information about your old commits.

**Comment by JonWBedard** [comment]
The commit message of the squashed commit is basically two commit messages?

**Review by aj062 — APPROVED** [review]

**Review by JonWBedard — APPROVED** [review]
